Autopilot/Common/Common.h

```diff
@@ -92,7 +92,8 @@ typedef struct _PMData { //54 Bytes
     int sp_Altitude; // Meters
     int heading;  //Degrees
     int sp_Heading; //Degrees
-    int batteryLevel;
+    int batteryLevel1;
+    int batteryLevel2;
```

Comment:
I believe it will map the voltage from the AN pin (0-3V3) to a 12 bit integer. So 0V=0, 3V3=111111111111